webservices/load_current_murs.py

```diff
@@ -49,6 +49,45 @@
     ;
 """
 
+CLOSE_DATE = """
+    SELECT event_date
+    FROM fecmur.calendar
+    WHERE case_id = %s
+    ORDER BY event_date DESC
+    LIMIT 1;
+"""
+
+OPEN_DATE = """
+    SELECT event_date
+    FROM fecmur.calendar
+    WHERE case_id = %s
+    ORDER BY event_date ASC
+    LIMIT 1;
+"""
+
+DISPOSITION_DATA = """
+    SELECT fecmur.event.event_name,
```

Comment:
I am guilty of silently introducing a convention of using uppercase for SQL keywords to make the entire SQL statement easy for human eyes. Feel free to ignore suggestion.